sources/Inputs/src/NetworkManager.cpp

```diff
@@ -21,14 +21,18 @@
 #include "Message.hpp"
 
 #include <DYNBusInterface.h>
+#include <DYNConverterInterface.h>
 #include <DYNDataInterfaceFactory.h>
 #include <DYNGeneratorInterface.h>
+#include <DYNHvdcLineInterface.h>
+#include <DYNLccConverterInterface.h>
 #include <DYNLineInterface.h>
 #include <DYNLoadInterface.h>
 #include <DYNNetworkInterface.h>
 #include <DYNThreeWTransformerInterface.h>
 #include <DYNTwoWTransformerInterface.h>
 #include <DYNVoltageLevelInterface.h>
+#include <DYNVscConverterInterface.h>
 #include <boost/make_shared.hpp>
```

Comment:
Remove all references to boost make_shared now that you are using std::shared_ptr